lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -28,6 +31,16 @@ def self.get_milestone(repository, release)
         return mile
       end
 
+      # Downloads a file from the given GitHub tag
```

Comment:
Actually even the doc description is incorrect (copy/pasted too quickly, right?)

Reply:
Uhhhh....you caught me 😅